### PR TITLE
Colibri to production

### PR DESCRIPTION
**Pull request by telphan**
@agherzan @floion

### CODE REVIEW

File: layers/meta-resin-toradex/recipes-bsp/u-boot/u-boot-toradex-fsl/0001-Split-defconfig-into-single-and-dual-version-of-the-.patch [hunk 1]
```diff
@@ -0,0 +1,46 @@
+From 19ac56196d5bd182e38b71e79291ad5b0912ed7a Mon Sep 17 00:00:00 2001
+From: Theodor Gherzan <theodor@resin.io>
+Date: Thu, 2 Jun 2016 22:03:20 +0200
+Subject: [PATCH] Split defconfig into single and dual version of the colibri
+ board
+
+Signed-off-by: Theodor Gherzan <theodor@resin.io>
```
Comment (agherzan) [inline layers/meta-resin-toradex/recipes-bsp/u-boot/u-boot-toradex-fsl/0001-Split-defconfig-into-single-and-dual-version-of-the-.patch]: Upstream status

File: layers/meta-resin-toradex/recipes-bsp/u-boot/u-boot-toradex-fsl_git.bbappend [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI_append = " \
```
Comment (agherzan) [inline layers/meta-resin-toradex/recipes-bsp/u-boot/u-boot-toradex-fsl_git.bbappend]: No need to split the string as long as it is one item in SRC_URI

File: layers/meta-resin-toradex/recipes-support/resin-init/files/resin-init-flasher-board [hunk 3]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
+
+if grep -q "Freescale i.MX6 Quad/DualLite" "/proc/cpuinfo" ; then
+    dd if=/boot/u-boot.imx-dual of=/dev/mmcblk0 conv=notrunc seek=1 bs=1024
+fi
```
Comment (agherzan) [inline layers/meta-resin-toradex/recipes-support/resin-init/files/resin-init-flasher-board]: This assumes you already have a single flashed. Can you do it for single too just for consistency?

Comment (telphan) [inline layers/meta-resin-toradex/recipes-support/resin-init/files/resin-init-flasher-board]: sure, will add.

**Comment by telphan** [comment]
Related to #9

**Comment by floion** [comment]
LGTM

File: layers/meta-resin-toradex/recipes-bsp/u-boot/u-boot-toradex-fsl/0001-Split-defconfig-into-single-and-dual-version-of-the-.patch [hunk 4]
```diff
@@ -0,0 +1,48 @@
+From 19ac56196d5bd182e38b71e79291ad5b0912ed7a Mon Sep 17 00:00:00 2001
+From: Theodor Gherzan <theodor@resin.io>
+Date: Thu, 2 Jun 2016 22:03:20 +0200
+Subject: [PATCH] Split defconfig into single and dual version of the colibri
+ board
+
+Upstream-Status: Pending
```
Comment (agherzan) [inline layers/meta-resin-toradex/recipes-bsp/u-boot/u-boot-toradex-fsl/0001-Split-defconfig-into-single-and-dual-version-of-the-.patch:7]: Why is this pending? Are you pending it to push upstream?

**Comment by agherzan** [comment]
LGTM

File: layers/meta-resin-toradex/recipes-bsp/u-boot/u-boot-toradex-fsl/0001-Split-defconfig-into-single-and-dual-version-of-the-.patch (hunk 4 above)
Comment (telphan) [inline layers/meta-resin-toradex/recipes-bsp/u-boot/u-boot-toradex-fsl/0001-Split-defconfig-into-single-and-dual-version-of-the-.patch:7]: Yes, I would like to push upstream, not exactly sure if this is something desired by the maintainer. Will raise a ticket anyways there.

**Pull request merged**